### PR TITLE
Document `pulumi:tags` config option.

### PR DESCRIPTION
**Pull request by ringods**
## Description

Fixes: #3263

## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [x] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

**Review by Frassle — APPROVED** [review]

File: themes/default/content/docs/concepts/config.md [hunk 1]
```diff
@@ -592,3 +592,20 @@ pulumi:disable-default-providers:
...
+
+```yaml
+  pulumi:tags:
+    company: "Some LLC"
+    team: Ops
+```
+
+Pulumi CLI only creates or updates tags which are listed in the config. If you remove a tag from the stack config, you have to remove it from the stack in Pulumi Cloud manually.
+
+The config sample above will be applied on the stack and become visible on Pulumi Cloud in the `Tags` section of your stack.
```
Comment (cnunciato) [inline themes/default/content/docs/concepts/config.md:609]:
```suggestion
Stack tags applied by Pulumi CLI are listed in the `Tags` section of the Overview tab:
```

File: themes/default/content/docs/concepts/config.md [hunk 2]
```diff
@@ -592,3 +592,20 @@ pulumi:disable-default-providers:
   - aws
   - kubernetes
 ```
+
+### `pulumi:tags`
+
+A list of [stack tags](/docs/concepts/stack/#stack-tags) which are read by the Pulumi CLI and automatically applied on the stack at
+every `pulumi up` or `pulumi refresh` action.
```
Comment (cnunciato) [inline themes/default/content/docs/concepts/config.md:599]: How does `refresh` actually work with stack tags? I tried adding a tag in Pulumi Cloud but not locally, and adding one locally but not in Pulumi Cloud, and in both cases, nothing happened when I ran `pulumi refresh`.

Comment (ringods) [reply themes/default/content/docs/concepts/config.md:599]: @Frassle shouldn't a `refresh` take the value from Pulumi Cloud and update the stack config file if the values differ?

File: themes/default/content/docs/concepts/config.md [hunk 3]
```diff
@@ -592,3 +592,20 @@ pulumi:disable-default-providers:
...
+
+```yaml
+  pulumi:tags:
+    company: "Some LLC"
+    team: Ops
```
[the remark this answers is not in the document]
Comment (ringods) [reply themes/default/content/docs/concepts/config.md:606]: Done!

**Pull request merged**